calico/felix/refcount.py

```diff
@@ -203,6 +214,10 @@ def _is_starting_or_live(self, obj_id):
 class RefHelper(object):
     """
     Helper class for a client of a ReferenceManager.  Manages the
+    # MD4 I think the first sentence here is confusing - this is not a helper class for most
+    # clients of this class - only those that want to wait for creation of multiple
+    # RefCountedActors.  I'm also not entirely sold on its existence, since it only has one
+    # user.  I'd be tempted to move it into profilerules.py at least.
```

Comment:
I had intended to use this in several places but we ended up with Endpoints no needing to actually get the reference to their profile.  I still think it's important and that it belongs here:

* It's important because this code is hard to get right.
* It belongs here because it knows about actor/ref count internals so it's more tightly coupled with those.  It's also completely generic and it's not that unlikely that LocalEndpoints will need to use one at some point.